adapters/repos/db/vector/hnsw/search.go

```diff
@@ -188,7 +188,7 @@ func (h *hnsw) acornParams(allowList helpers.AllowList) (bool, int) {
 			useAcorn = false
 		}
 		M = int(cacheSize / int64(max(1, allowListSize)))
-		M = min(M, 8)
+		M = min(M, 1)
```

Comment:
If we do this does it make sense to remove the `M` calculation and always use 1?